cob_3d_mapping_msgs/cob_3d_mapping_msgs_mapping_rules.yaml

```diff
@@ -0,0 +1,38 @@
+-
+  ros1_package_name: 'cob_3d_mapping_msgs'
+  ros1_message_name: 'CurvedPolygon'
+  ros2_package_name: 'cob_3d_mapping_msgs'
+  ros2_message_name: 'CurvedPolygon'
+  fields_1_to_2:
+    stamp: 'stamp'
+    ID: 'id'
+    parameter: 'parameter'
+    #polyline: 'polyline'
+    score: 'score'
+    polyline: 'polyline'
+    features: 'features'
+    energy: 'energy'
+    weight: 'weight'
+-
+  ros1_package_name: 'cob_3d_mapping_msgs'
+  ros1_message_name: 'Feature'
+  ros2_package_name: 'cob_3d_mapping_msgs'
+  ros2_message_name: 'Feature'
+  fields_1_to_2:
+    x: 'x'
+    y: 'y'
+    z: 'z'
+    ID: 'id'
+-
+  ros1_package_name: 'cob_3d_mapping_msgs'
+  ros1_message_name: 'SimilarityScore'
+  ros2_package_name: 'cob_3d_mapping_msgs'
+  ros2_message_name: 'SimilarityScore'
+  fields_1_to_2:
+    ID: 'id'
+    prob: 'prob'
+# add mapping rule for:
+#  - action/PlaneExtraction.action
+#  - action/TableObjectCluster.action
+#  - action/Trigger.action
+#  - srv/MoveToTable.srv
```

Comment:
LoyVanBeek please have a look
maybe you can help configure the missing mapping rules

Reply:
I tried mapping those in https://github.com/Deleh/cob_perception_common/pull/1 but doesn't compile. 
Not sure if that is my or `ros1_bridge`'s fault yet.

AFAIk, there is no docs on how to write mapping rules for Actions yet (service example is in https://github.com/mojin-robotics/ros1_bridge/blame/mojin-devel/doc/index.rst#L144), so it's 'Use the Source, Loy' :star2:  for that stuff for now. I looked at https://github.com/mojin-robotics/ros1_bridge/blob/mojin-devel/ros1_bridge/__init__.py#L622 for that.